tests/e2e/testcases/errhandling/suite.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package errhandling
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	"github.com/ava-labs/avalanche-cli/tests/e2e/utils"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+const (
+	subnetName = "doFailSubnet"
+)
+
+/*
+		The tests in this suite are meant to trigger some errors so that
+	  the UI will try to find errors in the log files.
+		However, one or more tests only trigger errors on timeouts,
+		which is why it should not be run in normal CI.
+
+		Therefore the tests are `Skip`ed, but can be enabled manually for testing
```

Comment:
how are they skipped? Just not imported in the main e22_test.go file?

Reply:
No by calling `Skip` in the test.